src/caffe/proto/caffe.proto

```diff
@@ -915,6 +916,11 @@ message PowerParameter {
   optional float shift = 3 [default = 0.0];
 }
 
+message PruningParameter {
+  // Pruning coefficient for deep compression
```

Comment:
It would be good to document what this parameter does here.  It looks like it's the fraction of weights to keep, sorted by absolute value?